### PR TITLE
F.1.1,2 Add and Edit Review of Course (Review part 1)

### PR DESCRIPTION
**Pull request by natalieloke**
# Feature

Use Case: 

F.1.1 | F | F.1 | Add Review to Course
F.1.2 | F | F.1 | Edit Review of Course

Feature ID : 8
Feature: Course Content

# Changelog:
- see commit log

## Checklist:

- [x] Merged latest develop
- [x] PR title makes sense

## Screenshots (where relevant):
<a href="https://www.loom.com/share/c4d41474e6804485ac24cfdc580426d7"> <p>Digi Dojo | Student add and edit a Course Review - Watch Video</p>  </a>

### CODE REVIEW

File: src/components/Mentorship/ListingDetails/MentorshipProfileHeader.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const MentorshipProfileHeader = () => {
   }, [])
 
   return (
-    <div className="row justify-content-between ">
+    <div className="row justify-content-between">
```
Comment (rawrrencee) [inline src/components/Mentorship/ListingDetails/MentorshipProfileHeader.js]: i think you might have a merge conflict once I merge my PR @ https://github.com/rawrrencee/is4103-frontend/pull/117

coz i am modifying this part as well to include the share button (but this page will be zhnged also in the near future)

File: src/components/Review/ReviewModal/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        onFinish={onSubmitReview}
+        onFinishFailed={onFinishFailed}
+      >
+        <Form.Item label="Rating" name="rating">
+          <Rate />
+        </Form.Item>
+
+        <Form.Item label="Review Comment" name="comment">
+          <TextArea
+            placeholder="Tell us more about what your experience."
```
Comment (rawrrencee) [inline src/components/Review/ReviewModal/index.js]: Tell us more about "--what--" your experience maybe? XD

**Review by rawrrencee — DISMISSED** [review]
might have merge conflict once I merge #117

File: src/components/Mentorship/ListingDetails/MentorshipProfileHeader.js (hunk 1 above)
Comment (natalieloke) [reply src/components/Mentorship/ListingDetails/MentorshipProfileHeader.js]: ok let me remove this fix

File: src/components/Review/ReviewModal/index.js (hunk 2 above)
Comment (natalieloke) [reply src/components/Review/ReviewModal/index.js]: oh right haha

**Review by rawrrencee — APPROVED** [review]
wokies

**Review by y1bs — APPROVED** [review]
Looks good to me

**Pull request merged**